shared/utils/uen-validation.ts

```diff
@@ -80,8 +113,18 @@ const VALID_ENTITY_TYPE_INDICATORS = new Set([
   // SNDGO
   'GA',
   'GB',
+
+  // Foreign Entities
+  'UF' 
 ])
 
+/**
+ * Helper to standardise format - uppercase and without whitespace  
+ * @param s String
+ * @returns True if string is numeric
+ */
+const standardise = (s: string): string => s.toUpperCase().trim()
```

Comment:
This might be better named to literally represent what it does with `upperCaseAndTrim`

Reply:
can!